src/EnergyPlus/PackagedTerminalHeatPump.cc

```diff
@@ -4186,6 +4186,15 @@ namespace PackagedTerminalHeatPump {
 		if ( CurZoneEqNum > 0 ) {
 			ZoneEqSizing( CurZoneEqNum ).OAVolFlow = max( PTUnit( PTUnitNum ).CoolOutAirVolFlow, PTUnit( PTUnitNum ).HeatOutAirVolFlow );
 			ZoneEqSizing( CurZoneEqNum ).AirVolFlow = max( PTUnit( PTUnitNum ).MaxCoolAirVolFlow, PTUnit( PTUnitNum ).MaxHeatAirVolFlow );
+			// save air flow rates for child object sizing
+			if( PTUnit( PTUnitNum ).MaxCoolAirVolFlow > 0.0 ) {
```

Comment:
The bools SystemAirFlow, CoolingAirFlow, and HeatingAirFlow are true whenever the values are calculated, but what is really needed is to know which one needs to be used.  So instead of a test like "PTUnit( PTUnitNum ).MaxCoolAirVolFlow > 0.0,"  I would look for conditions like  "PTUnit( PTUnitNum ).MaxCoolAirVolFlow > PTUnit( PTUnitNum ).MaxHeatAirVolFlow."   I just find it a bit confusing that both CoolingAirFlow and HeatingAirFlow can be true at the same time.  just a minor comment since this is already the way it is.